incubator/exporter-otlp/src/main/java/io/aklivity/zilla/runtime/exporter/otlp/internal/OltpExporterHandler.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.exporter.otlp.internal;
+
+import static io.aklivity.zilla.runtime.exporter.otlp.internal.config.OtlpSignalsConfig.Signals.METRICS;
+
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.LongFunction;
+
+import io.aklivity.zilla.runtime.engine.EngineConfiguration;
+import io.aklivity.zilla.runtime.engine.EngineContext;
+import io.aklivity.zilla.runtime.engine.config.AttributeConfig;
+import io.aklivity.zilla.runtime.engine.config.KindConfig;
+import io.aklivity.zilla.runtime.engine.exporter.ExporterHandler;
+import io.aklivity.zilla.runtime.engine.metrics.Collector;
+import io.aklivity.zilla.runtime.engine.metrics.reader.MetricsReader;
+import io.aklivity.zilla.runtime.exporter.otlp.internal.config.OtlpEndpointConfig;
+import io.aklivity.zilla.runtime.exporter.otlp.internal.config.OtlpExporterConfig;
+import io.aklivity.zilla.runtime.exporter.otlp.internal.config.OtlpSignalsConfig;
+import io.aklivity.zilla.runtime.exporter.otlp.internal.serializer.OtlpMetricsSerializer;
+
+public class OltpExporterHandler implements ExporterHandler
+{
+    private static final long RETRY_INTERVAL = Duration.ofSeconds(5).toMillis();
+    private static final long WARNING_INTERVAL = Duration.ofMinutes(5).toMillis();
```

Comment:
These should be in `OltpExporterConfiguration` for now instead of static constants.

Reply:
I understand they should be configurable in `zilla.yaml` so `OtlpOptionsConfig` and eventually `OtlpExporterConfig` picks them up. Is that right?

We shall modify the config to something like this? (All 3 `*Interval`s being optional and should have default values)
```
      type: otlp
      options:
        publishInterval: 30
        retryInterval: 10
        warningInterval: 300
        endpoint:
          http:
            metrics: http://localhost:4318/v1/metrics
```
Pls LMK.

Reply:
This was moved to `OltpConfiguration` after all, as discussed.